test/abi/Inputs/macOS/arm64/stdlib/baseline

```diff
@@ -7882,6 +7882,21 @@ _$ss20_SwiftNewtypeWrapperPss21_ObjectiveCBridgeable8RawValueRpzrlE024_condition
 _$ss20_SwiftNewtypeWrapperPss21_ObjectiveCBridgeable8RawValueRpzrlE026_unconditionallyBridgeFromD1CyxAD_01_D5CTypeQZSgFZ
 _$ss20_SwiftNewtypeWrapperPss21_ObjectiveCBridgeable8RawValueRpzrlE09_bridgeToD1CAD_01_D5CTypeQZyF
 _$ss20_SwiftNewtypeWrapperTL
+_$ss20__StaticArrayStorageC12_doNotCallMeAByt_tcfC
```

Comment:
You should be able to record these additions in `abi/macOS/stdlib.swift` instead of editing the baseline itself.